workflows/templates/ZWESECUR.vtl

```diff
@@ -400,6 +402,37 @@
   LISTGRP &HLQ.
   LISTDSD PREFIX(&HLQ.) ALL
 
+/* DEFINE ZOWE RESOURCE PROTECTION ................................. */
+
+/* - Defines new resource class for Zowe that protects access to     */
```

Comment:
mention that the class might already exist, in which case this step is not required

Reply:
I have added it.